src/OpenConext/ProfileBundle/Resources/views/Profile/overview.html.twig

```diff
@@ -11,19 +11,27 @@
         <table class="overview-table mdl-data-table">
             <thead>
             <tr>
-                <th>Attribute</th>
-                <th>Value</th>
+                <th>{{ 'profile.table.attribute_name'|trans }}</th>
+                <th>{{ 'profile.table.attribute_value'|trans }}</th>
             </tr>
             </thead>
             <tbody>
+            {% for attribute in attributes %}
```

Comment:
believe you could just use `app.user.attributes`, prevents having to pass the attributes as they are already available?